Permanent.xcodeproj/project.pbxproj

```diff
@@ -1137,6 +1167,7 @@
 				BC581A4D255A98DB00171D7A /* MediaRecorder.swift */,
 				BCC5501E255AB31800CA1574 /* FileHelper.swift */,
 				BD95ECC025D53E58009B443F /* ScreenLockManager.swift */,
+				5EADF80B262EDFC900D14E9C /* CustomViewFlowLayout.swift */,
```

Comment:
Can we rename this layout to "TagsCollectionViewLayout" or something similar to suggest it's used for tags?
And let's move it to the Views folder, instead of the Helpers folder